pkg/manifest/manifest.go

```diff
@@ -169,12 +169,18 @@ func (m *Manifest) ApplyDefaults() error {
 			m.Services[i].Scale.Count = ServiceScaleCount{Min: 1, Max: 1}
 		}
 
-		if m.Services[i].Scale.Cpu == 0 {
-			m.Services[i].Scale.Cpu = DefaultCpu
+		if m.Services[i].Scale.Gpu.Count == 0 {
```

Comment:
Just wondering why is that so. Couldn't we still set the default CPU and Memory values regardless? Or I guess that's a pattern when running GPU-specific pods?

Reply:
Yeah, there's a known issue/pattern where you should only specify GPU resources otherwise the nvidia-docker gets confused 🤷🏼‍♂️